services/static.js

```diff
@@ -128,6 +131,37 @@ var Static = {};
  *   });
  * const staticImageUrl = request.url();
  * // Now you can open staticImageUrl in a browser.
+ *
+ * @example
+ * // Filter all buildings that have a height value that is less than 300 meters
+ * const request = staticClient
+ *   .getStaticImage({
+ *     ownerId: 'mapbox',
+ *     styleId: 'streets-v11',
+ *     width: 200,
+ *     height: 300,
+ *     position: 'auto',
+ *     setfilter: [">","height",300],
```

Comment:
It's not currently possible to use the `'auto'` parameter with style parameters (you will receive a `422` response from the API), _unless_ you are also using an overlay. So I would recommend using an actual `0,0,0` type position for all of these examples.

Reply:
Thank you! This will be another good test!